mpop_service/facade.h

```diff
@@ -43,12 +36,12 @@ class Facade : public QObject
     int getOrCreateUser(const QString& rfidTag);
 
     /**
-     * @brief Retrieves all answers for a given user.
-     * @param userId The user ID of the user.
-     * @return A number of each question identifier.
-     * TODO: perhaps return a QVariant
-     */
-    QMap<QString, int> getUserAnswers(int userId);
+      * @brief Retrieves all answers for a given user.
+      * @param userId The user ID of the user.
+      * @return A number of each question identifier.
+      * TODO: perhaps return a QVariant
+      */
+     QMap<QString, int> getUserAnswers(int userId);
```

Comment:
We still don't want this indentation change for that function declaration.

Reply:
Ohh yeah fix it now sorry for delayed